src/rdkafka_coord.c

```diff
@@ -235,19 +235,35 @@ void rd_kafka_coord_req (rd_kafka_t *rk,
         creq->creq_replyq = replyq;
         creq->creq_resp_cb = resp_cb;
         creq->creq_reply_opaque = reply_opaque;
+        creq->creq_refcnt = 1;
```

Comment:
side note: on my todo list is to try and think through whether a refactor is possible anywhere that gets rid of refcounting (-> effectively multiple owners) in favor a single owner, which is simpler => more desirable if possible (even if there are some downsides, they may not be a big deal). probably not, but thought i'm mention the thought in case it gives you any ideas.